docs/source/pages/get_started.md

```diff
@@ -0,0 +1,183 @@
+# Get Started
+
+This is a brief introduction to getting started with DataShuttle, containing a subset of the full documentation.
+For more detail, please see the full documentation.
+
+This documentation gives examples both using the API (in the python console) or using the command line interface (in system terminal).
+
+## Installation
+
+DataShuttle is hosted on  [PyPI](https://pypi.org/project/datashuttle/) and can be installed with pip.
+
+`pip install datashuttle`
+
+Datashuttle required Rclone for data transfers. The easiest way to install Rclone is using [Miniconda](https://docs.conda.io/en/main/miniconda.html):
+
+```
+conda install -c conda-forge rclone
+```
+
+See [the Rclone website](https://rclone.org/install/) for alternative installation methods.
+
+## Initial Setup - Set Configurations
+
+To get started, import DataShuttle and initialise the class with the project name. If using the command
+line this step is not necessary.
+
+```
+from datashuttle.datashuttle import DataShuttle
+
+project = DataShuttle("my_project")
+```
+
+DataShuttle helps to manage and transfer a project with many "local" machines all connected to a central "remote" machine.
+DataShuttle requires a path to the local project directory (this is typically empty on first use and should include the name of the project
+e.g. local_path="/path/to/my_project"). Also, remote project directory and method to connect to the remote project (either "local_filesystem" or
+"ssh").
+
+It is also required to specify the data types (e.g. behav, ephys, funcimg, histology) used on the local PC. If data_type flags are not set to True, it will not be possible
```

Comment:
As above, the user may not understand what exactly the data types are, so they should look through swc-blueprint first.